m3.yml

```diff
@@ -14,47 +14,47 @@
 #   date_modified:  the date the profile was last altered
 #   type:  type of thing does the profile describe
 #   version:  version of the profile
- 
+
 profile:
   responsibility: https://wiki.duraspace.org/display/samvera/Samvera+Metadata+Interest+Group
   responsibility_statement: Samvera Metadata Interest Group
   date_modified: 2019-07-03
   type: metadata models
   version: 0.8
-  
+
 #---------------------------------------------------------
 # Mapping Definitions
 #---------------------------------------------------------
 # Definition of the mappings to different services or target schemas referenced in the profile.
 #
-# mappings: 
+# mappings:
 # - prefix: the abbreviation used to indicate the mapping target
 #   name: a fuller name of the mapping
- 
+
 mappings:
- - dpla:
```

Comment:
The difference in how this is parsed:
```
 "mappings"=>
  [{"dpla"=>nil, "name"=>"Digital Public Library of America"},
   {"datacite"=>nil, "name"=>"DataCite"},
   {"dc"=>nil, "name"=>"Dublin Core"}],
```
versus
```
 "mappings"=>
  {"dpla"=>{"name"=>"Digital Public Library of America"}, "datacite"=>{"name"=>"DataCite"}, "dc"=>{"name"=>"Dublin Core"}},
```
I assume we want the latter.

Reply:
yes, definitely the second one.